pensjon-brevbaker/src/test/kotlin/no/nav/pensjon/brev/api/TemplateResourceITest.kt

```diff
@@ -1,51 +1,42 @@
 package no.nav.pensjon.brev.api
 
-import io.mockk.mockk
 import no.nav.pensjon.brev.*
 import no.nav.pensjon.brev.api.model.*
-import no.nav.pensjon.brev.api.model.maler.BrevbakerBrevdata
 import no.nav.pensjon.brev.api.model.maler.Brevkode
-import no.nav.pensjon.brev.maler.ProductionTemplates
+import no.nav.pensjon.brev.fixtures.createLetterExampleDto
+import no.nav.pensjon.brev.maler.example.LetterExample
 import no.nav.pensjon.brev.template.LetterTemplate
 import no.nav.pensjon.brevbaker.api.model.LanguageCode
 import org.junit.jupiter.api.*
-import kotlin.reflect.KClass
 
 @Tag(TestTags.INTEGRATION_TEST)
 class TemplateResourceITest {
-    private val templateResource = TemplateResource("autobrev", ProductionTemplates.autobrev, mockk())
 
     @Test
     fun `can render and compile a template`() {
-        requestTemplates()
-            .associateWith { templateResource.getTemplate(it)!! }
-            .firstNotNullOf { it }
-            .let { testTemplate(it.key, it.value.template) }
+        testTemplate(LetterExample.kode, LetterExample.template)
     }
 
     private fun testTemplate(kode: Brevkode.AutoBrev, template: LetterTemplate<*, *>?) {
         if (template == null) {
             fail { "TemplateResource.getTemplates() returned a template name that doesnt exist: $kode" }
         }
-        val argument = createArgument(template.letterDataType)
+        val argument = createLetterExampleDto()
         try {
-            val rendered = requestLetter(
-                BestillBrevRequest(
-                    kode = kode,
-                    letterData = argument,
-                    felles = Fixtures.felles.copy(signerendeSaksbehandlere = null),
-                    language = LanguageCode.BOKMAL
+            testBrevbakerApp { client ->
```

Comment:
Tester tagget med INTEGRATION_TEST blir kjørt av :integrationTest gradle-target. Da har vi brevbaker kjørende, så det er egentlig ikke nødvendig å spinne opp en testBrevbakerApp. Er tanken din å bytte vekk fra den kjørende brevbaker-containeren?

Reply:
Ser ikkje heilt korleis unngå det - poenget er å sørge for at LetterExample og EksempelbrevRedigerbart hamnar i lista over malar tilgjengeleg (i tillegg til eksisterande ProductionTemplates og EtterlatteTemplates, altså).